tests/gdimagesetpixel/gdeffectdiff.c

```diff
@@ -0,0 +1,33 @@
+#include "gd.h"
```

Comment:
please add a proper file header: license, copyright, and text explaining the test goals.